### ISSUE TITLE
[Enhancement]: aws_organizations_policy

### ISSUE THREAD

**Issue by Gianluca755** [hashicorp/terraform-provider-aws#38788]
### Description

The provider checks at plan time if the SCP is above the limit of 5120 characters.

### Affected Resource(s) and/or Data Source(s)

aws_organizations_policy

### Would you like to implement a fix?

None

**Comment by CorinneVerheyde** [comment]
Would definitely be extremely usefull, and for any type of policy (user policy, group policy, role policy, etc...).

**Comment by ivankatliarchuk** [comment]
Agree. This is the docs https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies.html

aws_organizations_policy does not support pre-conditions, so hard to validate as well

**Comment by mdgm88** [comment]
The SCP size limit is now 10,240 characters: https://aws.amazon.com/about-aws/whats-new/2026/05/aws-organizations-increased-scp-quotas/
